closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * This function is called when the length of the repeating digits is at least
```

Comment:
It doesn't have to be a huge paragraph, I just want a succinct description of what exactly this method does. It should be as simple as possible while getting the point across. Like "Takes a `foo` does `bar` then returns a final `baz` except if `blah` happens".